controllers/add_pods.go

```diff
@@ -35,7 +37,7 @@ type AddPods struct{}
 
 // Reconcile runs the reconciler's work.
 func (a AddPods) Reconcile(r *FoundationDBClusterReconciler, context ctx.Context, cluster *fdbtypes.FoundationDBCluster) *Requeue {
```

Comment:
Should we consider moving the subreconcilers into the internal package? Having them in the public package means that we wouldn't be able to fully remove one without a major release.

Reply:
If we moved them into the internal package we could also look into creating subpackages for different reconcilers to keep the code more organized.

Reply:
Sounds good, I would prefer to open a follow up issue for that and move the subreconcilers in another PR. This PR is already touching too many files :) https://github.com/FoundationDB/fdb-kubernetes-operator/issues/827